typedoc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "mode": "modules",
+  "out": "./docs/autogen",
+  "json": "./docs/autogen/output.json",
+  "theme": "default",
+  "ignoreCompilerErrors": "true",
```

Comment:
I'll remove it, it was useful for testing. Individual components which didn't have typings in the same folder (`src/lib/button`).